posthog/session_recordings/queries/session_recording_list_from_filters.py

```diff
@@ -334,6 +360,53 @@ def _where_predicates(self) -> ast.Expr:
         )
 
 
+class CohortPropertyGroupsSubQuery:
+    _team: Team
+    _filter: SessionRecordingsFilter
+    _ttl_days: int
+
+    raw_cohort_to_distinct_id = """
+    select distinct_id
+    from person_distinct_ids
+    where person_id IN (
+        SELECT person_id
+        FROM cohort_people
```

Comment:
at first i went to the static/dynamic cohort tables, but they ended up querying this cohort_people table and doing nothing else, so let's go straight to cohort people